pwem/objects/data.py

```diff
@@ -870,8 +872,7 @@ def __init__(self, filename=None, pseudoatoms=False, **kwargs):
 class EMSet(Set, EMObject):
```

Comment:
What about declaring here:
USE_CREATE_COPY_FOR_SUBSET = False

And avoiding hasAttr in the protocol batch? just  `if modifiedSet.USE_CREATE_COPY_FOR_SUBSET` ?

Reply:
I don't like it since it is something that should be deprecated, and it is not expected to be used in other places (remember that subset logic does not necessarily be the one with the gui/batch one). I would keep the getattr with default False there...and keep the EMSet class more clean.